newrelic/csec-node-agent
# feat: Functionality to report API endpoints of the application

## Description

Functionality to report API endpoints of the application. Detect all routes and generates "sec-application-url-mapping" event

## How to Test

Run application with IAST enabled and check for generated event for API end point at startup.

## Related Issues

Fixes: [NR-222180](https://new-relic.atlassian.net/browse/NR-222180)
